scripts/foundry/Constants.s.sol

```diff
@@ -6,6 +6,18 @@ import { Safe, Enum } from "safe/Safe.sol";
 import { ITransmuter } from "transmuter/interfaces/ITransmuter.sol";
 import { IAgToken } from "borrow/interfaces/IAgToken.sol";
 
+interface IVaultManagerGovernance {
```

Comment:
Normally you can import them from the borrow repo

Reply:
we hadn't defined it in the borrow repo